libs/types/src/cdf/election-results-reporting/convert.test.ts

```diff
@@ -241,6 +280,56 @@ describe('getManualResultsFromErrElectionResults', () => {
     expect(results.ok()).toEqual(expected);
   });
 
+  test('yes no ContestSelections defaults "yes" to first ContestSelection', () => {
```

Comment:
FYI @adghayes we discussed avoiding this behavior in a previous PR, but even our fixture election couldn't be matched on `yes` and `no` strings (IIUC it uses "For" and "Against"). I added this in since it's seeming more like a necessary feature, but LMK if you have any concerns.